hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemAttributes.java

```diff
@@ -91,13 +90,6 @@ public void testSetGetXAttrReplace() throws Exception {
         .containsExactly(attributeValue2);
   }
 
-  @Test
-  public void testGetSetXAttrOnRoot() throws Exception {
-    AzureBlobFileSystem fs = getFileSystem();
-    final Path testPath = new Path("/");
-    testGetSetXAttrHelper(fs, testPath);
```

Comment:
To prevent revert add of getFileSystemProperties and setFileSystemProperties for getXAttr / setXAttr in future, should we not remove this method, and additionally assert that the getFileSystemProperties and setFileSystemProperties of store are not called.

Reply:
Lets add an assert that this should throw 4xx error.

Reply:
Reason for this comment:
1. Lets say getFsProperties / setFSProperties are added again in future. This test will raise an CI issue on that patch.

Reply:
Makes sense.
Added back the negative test.